src/common/utils.tsx

```diff
@@ -79,3 +80,17 @@ export const downloadJsonDataFile = (jsonTemplate: any): void => {
   const jsonBlob = new Blob([jsonData], { type: "text/json;charset=utf-8" });
   saveAs(jsonBlob, "sample-data.json");
 };
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const isAwsKmsSignerOption = (option: any): option is AwsKmwSignerOption => {
```

Comment:
where is this being used?

Reply:
Currently, it isn't being used. i was thinking of keeping it for future usage if required, should I remove it instead?

Reply:
code that isnt being used will rot, so better to remove it; this is just the logic from `oa-cli` right?

Reply:
yes, i got the logic from oa-cli. okay will remove it